ingest/ingest_pipeline.py

```diff
@@ -102,6 +103,7 @@
     from .clusters import Clusters
     from .expression_files.dense_ingestor import DenseIngestor
     from .expression_files.mtx import MTXIngestor
+    from .h5ad import H5adIngestor
```

Comment:
Out-of-scope here, but maybe worthwhile as part of the larger "simplify Ingest Pipeline" initiative this quarter:

This kind of redundant `import` block is an obvious maintainability issue in Ingest Pipeline.  A code comment above notes that it exists due to code in `single_cell_portal` repo, i.e. the public SCP CLI / `manage_study.py`.  But I suspect there's a better solution, e.g. either:

1. Simply declare the public SCP CLI unsupported (I recall analytics that showed essentially zero use, prompting indefinite postponement of related work) and delete the redundant imports, or
2. Implement a way to declare the imports once, while still satisfying that public SCP CLI use case (and testing it)

Reply:
Will add this to the list of "simplify Ingest Pipeline" discussion topics